source/runtime.cpp

```diff
@@ -110,6 +118,8 @@ namespace reshade
 
 		_is_initialized = true;
 		_last_reload_time = std::chrono::high_resolution_clock::now();
+		_init_game_list(_game_list);
+		_host_process_name = _get_host_app();
```

Comment:
ReShade should be kept generic, so these changes shouldn't go into the official repository.

Reply:
OK, will keep it generic.